src/data/error.rs

```diff
@@ -114,6 +114,27 @@ pub enum SemanticError {
         current: u64,
     },
 
+    #[error("unreachable statement")]
+    UnreachableStatement,
+
+    #[error("redeclaration of label {0}")]
+    LabelRedeclaration(cranelift::prelude::Ebb),
+
+    #[error("use of undeclared label {0}")]
+    UndeclaredLabel(crate::intern::InternedStr),
+
+    #[error("{}case outside of switch statement", if *(.is_default) { "default " } else { "" })]
+    CaseOutsideSwitch { is_default: bool },
+
+    #[error("cannot have multiple default cases in a switch statement")]
+    MultipleDefaultCase,
```

Comment:
This is part of a more general error which is "case repeated in switch statement". We don't catch that one currently but we will soon: https://github.com/jyn514/rcc/issues/221

Reply:
We can change the error once the other duplicate cases are detected

Reply:
Can you change the name to DuplicateCase? It looks good after that.

Reply:
I'll do this myself when I fix https://github.com/jyn514/rcc/issues/221